fdbserver/include/fdbserver/DDTxnProcessor.h

```diff
@@ -23,7 +23,13 @@
 
 #include "fdbserver/Knobs.h"
 #include "fdbserver/DataDistribution.actor.h"
```

Comment:
Doesn't the inclusion of a `.actor.h` map this have to be a `.actor.h` and include the actorcompiler header? Maybe not now, but could it result in strange compile-time problem later?

Reply:
#7557 address the circular header